model/UserManager.php

```diff
@@ -9,7 +9,13 @@ public function getUser(string $pseudo): object
         $query = $this->db->prepare('SELECT * FROM users WHERE pseudo = ?');
         $query->execute([$pseudo]);
 
-        return new User($query->fetch());
+        if (is_bool($query->fetch())) {
+            throw new \Exception("Ce pseudo n'existe pas.");
+        } else {
+            var_dump("test");
```

Comment:
`var_dump`s should never go in production ;)

Reply:
oups^^'